<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1280</width>
    <height>720</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1280</width>
    <height>720</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>1261</width>
      <height>681</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Tab 1</string>
     </attribute>
     <widget class="QPushButton" name="deleteButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>470</y>
        <width>211</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>Delete Car</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="deleteLine">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>430</y>
        <width>211</width>
        <height>28</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="countLine">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>310</y>
        <width>211</width>
        <height>28</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="addButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>220</y>
        <width>211</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>Add Car</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="vinLine">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>180</y>
        <width>171</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>260</y>
        <width>211</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>count cars with lower year of 
production than:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>100</y>
        <width>51</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>model:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>180</y>
        <width>41</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>VIM:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="countButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>350</y>
        <width>211</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>Count Cars</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>400</y>
        <width>211</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>type VIM number:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="brandLine">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>60</y>
        <width>151</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>211</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>type car data:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="productionLine">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>140</y>
        <width>121</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLineEdit" name="searchLine">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>20</y>
        <width>911</width>
        <height>28</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="modelLine">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>100</y>
        <width>151</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="sortY">
      <property name="geometry">
       <rect>
        <x>1170</x>
        <y>20</y>
        <width>31</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="assets.qrc">
        <normaloff>:/icons/assets/iconmonstr-sort-19.svg</normaloff>:/icons/assets/iconmonstr-sort-19.svg</iconset>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>41</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>brand:</string>
      </property>
     </widget>
     <widget class="QListWidget" name="listWidget">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>60</y>
        <width>991</width>
        <height>571</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>140</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>production:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="sortA">
      <property name="geometry">
       <rect>
        <x>1210</x>
        <y>20</y>
        <width>31</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="assets.qrc">
        <normaloff>:/icons/assets/iconmonstr-sort-14.svg</normaloff>:/icons/assets/iconmonstr-sort-14.svg</iconset>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Tab 2</string>
     </attribute>
     <widget class="QListWidget" name="listWidget_2">
      <property name="geometry">
       <rect>
        <x>720</x>
        <y>250</y>
        <width>256</width>
        <height>192</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="assets.qrc"/>
 </resources>
 <connections/>
</ui>
